python/SeanZvvHbb13TeVSteps/README.md

```diff
@@ -0,0 +1,71 @@
+
+The steps to run are located in [SeanZvvHbb13TeVSteps/](../SeanZvvHbb13TeVSteps). All of the commands and steps mentioned should be run from within the Xbb/python/ directory.
+
+###1. Set Up the Working Environment
+
+Set the options in the **[Configuration]** section and **Working Environment** subsection of [**paths.ini**](../SeanZvvHbb13TeVconfig/paths.ini).
+
+While there are eight cores available on LXPLUS, it is better to lower the number of processes when testing and debugging. From browsing other users' configuration files, I've seen directory paths end with and without a trailing slash. I've adopted the convention to omit the trailing slash.
+  
+###2. **Prepare the Samples**
+  
+The ntuples for the samples are stored at INFN's Tier2 computing center in Pisa (grazie a Andrea). The large size of these ntuples makes them unwieldy to repeatedly process when optimizing an analysis strategy, so they are first skimmed with a selection cut to reduce their size and copied locally for convenience.
+
+Set the options in the **Preparation Step** subsection of [**paths.ini**](../SeanZvvHbb13TeVconfig/paths.ini).
+
+Set the options in the **[General]** section of [**samples_nosplit.ini**](../SeanZvvHbb13TeVconfig/samples_nosplit.ini) and, below the **[Samples_running]** section, create or modify the sample sections.
+
+Run the [das_query.py](../das_query.py) script, which will query the [CMS Data Aggregation System](https://cmsweb.cern.ch/das/) (DAS) to locate the sample (or dataset) and its corresponding ntuples (or files).
+```bash
+./das_query.py SeanZvvHbb13TeVconfig
+```
+The script should finish in under ten minutes. There should be a new directory within SeanZvvHbb13TeVconfig/ which contains files filled with logical file names (LFNs) for each sample.
+
+To skim and copy a sample, run the [prep.py](../prep.py) script.
+```bash
+# Create a valid GRID proxy to access remote files.
+voms-proxy-init -voms cms
+./prep.py SeanZvvHbb13TeVconfig sampleName
+```
+The [TreeCopier](../utils/treecopier.py) class will skim and copy ntuples using eight processes. The time it takes to finish varies based on the number of ntuples to prepare, the skimming selection, and the connection to the remote server.
+
+Otherwise, run the [Step0_prep.sh](Step0_prep.sh) script to skim and copy all the samples.
+```bash
+# Create a valid GRID proxy to access remote files.
+voms-proxy-init -voms cms
+bash SeanZvvHbb13TeVSteps/Step0_prep.sh
+```
+This will definitely take a lot of time to finish, on the order of half a day.
+
+If you want to run this script from within a screen or tmux session so you don't have to worry about disconnecting from LXPLUS, see this [article](https://cern.service-now.com/service-portal/article.do?n=KB0002408). Here's an example bash function that I use to create a tmux session with kerberos authentication renewal, where the first command line argument is the name given to the session.
+```bash
+ktmux () {
```

Comment:
do you know about screen?
https://www.rackaid.com/blog/linux-screen-tutorial-and-how-to/
basic commands:
screen (creates a screen)
CTRL+A+D (detaches the screen)
screen -r (recovers the screen)
exit (typing it in the screen, kills it)